qemu/tests/cfg/blockdev_full_backup_nonexist_target.cfg

```diff
@@ -0,0 +1,24 @@
+- blockdev_full_backup_nonexist_target:
+    only Linux
+    virt_test_type = qemu
+    type = blockdev_full_backup_nonexist_target
+    kill_vm_on_error = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    images += " src1"
+    start_vm = no
+    not_preprocess = yes
+    storage_pool = default
+    image_size_src1 = 200M
+    image_name_src1 = "src1"
+    force_create_image_src1 = yes
+    force_remove_image_src1 = yes
+    sync = full
+    source_images = src1
+    target_images = dst1
+    image_size_dst1 = 200M
+    image_name_dst1 = "dst1"
+    image_format_dst1 = qcow2
+    rebase_mode = unsafe
+    qmp_error_msg = "Cannot find device='drive_dst1' nor node-name='drive_dst1'"
+    qmp_error_msg += | "Cannot find device=drive_dst1 nor node-name=drive_dst1"
```

Comment:
Just fyi.
what about qmp_error_msg += "|Cannot find xxx", in the python way, please check the debug log to see if qmp_error_msg is well set, if it's fine, either is OK for me

Reply:
qmp_error_msg += "|Cannot find xxx" is not correct, we can't match it. The correct pattern is qmp_error_msg +=  | "Cannot find xxx"

Reply:
I am a little confused, I wrote a test with your format, and got the following string:
error_msg = because it is opened read-only| "you are not lzc"
So in your case, you might get the string:
qmp_error_msg = Cannot find xxx| "Canot findxxxx"
Can you please make a double check from your debug log?

Reply:
Yes, zhenchao, you're right. The qmp_error_msg = Cannot find xxx|  "Canot findxxxx" in my code.
 I add a redudant " " in the cfg, so match for Cannot find device=drive_dst1 nor node-name=drive_dst1| Cannot find device='drive_dst1' nor node-name='drive_dst1' failed.